templates/CRM/Contribute/Form/Contribution/OnBehalfOf.tpl

```diff
@@ -35,7 +35,7 @@
           </div>
         </div>
       {/if}
-      {include file="CRM/UF/Form/Block.tpl" fields=$onBehalfOfFields mode=8 prefix='onbehalf'}
+      {include file="CRM/UF/Form/Block.tpl" fields=$onBehalfOfFields mode=8 prefix='onbehalf' hideFieldset=false}
```

Comment:
It looks like hideFieldset needs to be true here to avoid the duplicate fieldset/title. Or at least that's one way to do it.

Reply:
@demeritcowboy maybe that explains the mystery of why the fieldset is hideable... (fix pushed)